rfcs/20210624-persistent-cache/README.md

```diff
@@ -0,0 +1,259 @@
+# Persistent Cache in oneDNN
+
+## Motivation
+
+oneDNN provides only in-memory primitive cache capabilities which is not
+sufficient for the scenarios when time of the first model run is significant,
+for example in the case of GPU primitives. An average GPU primitive creation time
+is 400ms comparing to less than a millisecond for CPU primitives. The original
```

Comment:
Does this issue affect DPC++ kernels as well or building from SPIR-V is not that bad?

Reply:
As far as I remember creating a DPC++ kernel was even more expensive than 400ms when I was working on enabling DPC++ kernels in oneDNN. Maybe it's not the case now. But anyway, turning a SPIR-V to a device binary is still quite expensive operation because it involves an optimizer which generally takes more time than lowering a DSL to SPIR-V.

The DPC++ compiler has its own persistent cache but it doesn't work for interop API, in other words it doesn't work for us at all because we use nGEN and OpenCL to get the binary and then create the corresponding SYCL objects. Although we can probably ask them to extend it to the interop API but it won't really help OpenVINO.

Reply:
makes sense, thanks for the explanation